### DISCUSSION TITLE
Replies of replies

### DISCUSSION THREAD

**Discussion by TSolo315** [Radi85/Comment#182] [Ideas]
I think allowing for replies of replies could be useful in many situations.

At the very least the fact that nested replies are not implemented should be added to the documentation (I spent half an hour trying to figure out why it wasn't working, thinking it was a bug.)

**Comment by abhiabhi94** [comment]
> At the very least the fact that nested replies are not implemented should be added to the documentation

It is an open-source application, you are complete welcome to add such a thing. Maybe you can mention it in the [`introduction.rst`](https://github.com/Radi85/Comment/blob/develop/docs/source/introduction.rst) somewhere.

**Reply by Radi85** [reply]
> At the very least the fact that nested replies are not implemented should be added to the documentation (I spent half an hour trying to figure out why it wasn't working, thinking it was a bug.)

Really? I believe the purpose of the docs is to guide the users on how to use the **implemented** features not the way around.

**Reply by abhiabhi94** [reply]
I think we can have a section which lists the limitation of the current application. This might seem to be informative for people. We can also mention people to open a potential pull request if they want to add a feature :)

**Reply by TSolo315** [reply]
> Really? I believe the purpose of the docs is to guide the users on how to use the **implemented** features not the way around.

I mean, I'm not suggesting we explain why the app doesn't allow you to order a pizza.

When you choose a reply as a parent comment there is no error and reply comments also return a replies field (that is always empty.)

It's not intuitive that the feature is not implemented. My first instinct was that I messed something up, and after failing to find anything my second instinct was to check the documentation to see if I missed something there.

I'm suggesting clarification of a feature's capabilities.

On a side note, removing the check and allowing for replies of replies does actually work when using the api. I'm assuming it breaks one of the other features? Like subscribing to a thread?

**Reply by Radi85** [reply]
That makes sense, I would suggest to emphasise in the docs that the current implementation is supporting a single level of replies.

However, the good news is that the replies nested levels can be increased with the current comment model structure. As a first sight I see the complexity will be with updating the UI and API to adopt this feature.
